internal/proto/query_coord.proto

```diff
@@ -389,3 +414,83 @@ message SealedSegmentsChangeInfo {
   common.MsgBase base = 1;
   repeated SegmentChangeInfo infos = 2;
 }
+
+message GetDataDistributionRequest {
+  common.MsgBase base = 1;
+}
+
+message GetDataDistributionResponse {
+  common.Status status = 1;
+  int64 nodeID = 2;
+  repeated int64 growing_segmentIDs = 3;
+  repeated SegmentVersionInfo segments = 4;
+  repeated ChannelVersionInfo channels = 5;
+  repeated LeaderView leaderViews = 6;
+}
+
+message LeaderView {
+  int64 collection = 1;
+  string channel = 2;
+  map<int64, int64> segmentNodePairs = 3;
+}
+
+
+message SegmentVersionInfo {
+  int64 ID = 1;
+  int64 collection = 2;
+  int64 partition = 3;
+  string channel = 4;
+  int64 version = 5;
+}
+
+message ChannelVersionInfo {
+  string channel = 1;
+  int64 collection = 2;
+  int64 version = 3;
+}
+
+enum LoadStatus {
+  Invalid = 0;
+  Loading = 1;
+  Loaded = 2;
+}
+
+message CollectionLoadInfo {
+  int64 collectionID = 1;
+  repeated int64 released_partitions = 2;
+  int32 replica_number = 3;
+  LoadStatus status = 4;
+}
+
+message PartitionLoadInfo {
+  int64 collectionID = 1;
+  int64 partitionID = 2;
+  int32 replica_number = 3;
+  LoadStatus status = 4;
+}
+
+message Replica {
+  int64 ID = 1;
+  int64 collectionID = 2;
+  repeated int64 nodes = 3;
+}
+
+enum SyncType {
+  Remove = 0;
+  Set = 1;
+}
+
+message SyncAction {
+  SyncType type = 1;
+  int64 partitionID = 2;
+  int64 segmentID = 3;
+  int64 nodeID = 4;
+}
+
```

Comment:
Uniform naming conventions

sometimes is camelcase sometimes is snakecase currently

Reply:
@jiaoew1991  this naming hacks protoc, `nodeID` generates `NodeID`, but `node_id` generates `NodeId`, which the latter can't pass CI, for the other language, snakecase generates the correct name.

Reply:
`NodeId` generated by proto will not affect code check.